support-frontend/assets/helpers/logger.js

```diff
@@ -1,39 +1,40 @@
 // @flow
 // import Raven from 'raven-js';
```

Comment:
can we delete this commented import?

Reply:
thanks yes maybe that was left over from when I made sentry load async.
Although having thought about that particular change a bit more, I'm not sure whether loading it async would collect errors that happen early in the page load!